app/index.js

```diff
@@ -119,6 +119,21 @@ function initialize (config) {
 
   }
 
+  // Redirect rules
+  if (config.redirects) {
+    app.use(function (req, res, next) {
+      var redirectUrl = config.redirects[req.baseUrl + req.path];
```

Comment:
we should add some comments here to explain what/why we are doing this (e.g. allow redirects with query string)